platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,21 +42,8 @@
     <bean id="security" class="org.codice.ddf.security.common.Security"
           factory-method="getInstance"/>
 
-    <!-- Base URI for DDF home -->
-    <bean id="ddfHomeFile" class="java.io.File">
-        <argument value="${ddf.home}"/>
-    </bean>
-
-    <bean id="ddfHomeUri" class="java.net.URI" factory-ref="ddfHomeFile" factory-method="toURI"/>
-
-    <!-- Default Directory Export Path -->
-    <bean id="defaultExportDirectoryUri" class="java.nio.file.Uri" factory-ref="ddfHomeUri"
-          factory-method="resolve">
+    <bean id="defaultExportDirectoryPath" class="java.io.File">
         <argument value="${ddf.home}/etc/exported"/>
```

Comment:
I understand this is a workaround, but we need to eventually fix this. We are relying on `java.io.File` to handle this malformed path which is not ok. We need to find a good solution and fix this for all our blueprints.